FSW/src/headers/config.hpp

```diff
@@ -69,13 +80,45 @@ static Event saveBufferEvent;
 static TimedEvent sunriseTimerEvent(WINDOW_LENGTH_MSEC);
 static TimedEvent sweepTimeoutEvent(SWEEP_TIMEOUT_MSEC);
 
+// FUTURE TEAMS: this event is invoked when the ADCS should switch its sweep direction
+//   so link your ADCS module with this event to tell it when to switch direction 
+//      look at checkSweepChange() in timing.cpp for more info
+static Event sweepDirectionChangeEvent; 
+
+
 /* - - - - - - Command Handling Module - - - - - - */
 const int COMMAND_QUEUE_SIZE = 100;     // maximum number of commands the command queue can store.
 const int SERIAL_TIMEOUT_MSEC = 50;     // milliseconds, maximum time to wait for serial input
 
+
 /* - - - - - - Fault Mitigation Module - - - - - - */
-const int WD_RESET_INTERVAL = 100;  // watchdog feeding interval, ms
-const int WD_PULSE_DUR = 10;        // watchdog reset signal duration, MICROSECONDS!!!
+const int WD_RESET_INTERVAL_MSEC = 100;     // milliseconds, watchdog feeding interval
+const int WD_PULSE_DUR_MICROSEC = 10;       // microseconds, watchdog reset signal duration
+
+
+/* - - - - - - Housekeeping Module - - - - - - */
+const int HK_SAMPLES_TO_KEEP = 100;   // number of previous housekeeping samples to keep

```

Comment:
How much time would this cover? I didn't do the math. I imagine we should keep _all_ the housekeeping data between downlink periods for debugging. Data science might want to link temperatures with the data on the ground.

Reply:
Very good point. It should be somewhere around ~5000 elements large depending on the orbit. With six floats per sample all the housekeeping data will be about 0.5Mbits